### PR TITLE
add airing status to library entries

### PR DESCRIPTION
**Pull request by matthewdias**
Changes proposed in this pull request:

add airing status tags to library entries

### CODE REVIEW

File: app/components/users/library/library-entry.js
Comment (matthewdias) [inline app/components/users/library/library-entry.js]: @vevix is there a place we can put this method so that both `media-information` and `library-entry` components can share it?

File: app/templates/components/users/library/library-entry.hbs
Comment (trmcnvn) [inline app/templates/components/users/library/library-entry.hbs]: Could you move this up to be a child of `entry-indicators`.

File: app/components/users/library/library-entry.js
Comment (trmcnvn) [reply app/components/users/library/library-entry.js]: We can move it to `models/media.js` with some minor changes.

```javascript
/**
 * Returns the translation key for the current airing status of this media.
 *
 * @returns {string} Translation Key
 */
airingStatus: computed('{startDate,endDate,unitCount}', function() {
  const start = get(this, 'startDate');
  const end = get(this, 'endDate');
  if (start && (start.isBefore() || start.isSame())) {
    const isOneDay = get(this, 'unitCount') === 1;
    const isPastDay = end && (end.isBefore() || end.isSame());
    return (isOneDay || isPastDay) ? 'finished' : 'current';
  }
  return 'nya';
}).readOnly()
```

I'm not sure if `{..,..}` will work without a context, If it doesn't change it to:

`computed('startDate', 'endDate', 'unitCount', ...`

**Review by trmcnvn — CHANGES_REQUESTED** [review]
:heart:

File: app/templates/components/users/library/library-entry.hbs
Comment (matthewdias) [reply app/templates/components/users/library/library-entry.hbs]: ah, thats what that was for

**Comment by matthewdias** [comment]
hold off on this, i still need to implement `Upcoming`, but ran into some issues so I just pushed what I had for now.

**Comment by matthewdias** [comment]
@vevix should be good to go now.

File: app/models/media.js
Comment (trmcnvn) [inline app/models/media.js]: What's the reasoning behind moving these here? It seems they are only used in the single place.

Comment (matthewdias) [reply app/models/media.js]: ah whoops i was initially going to calculate upcoming based on season, and forgot to move them back when i switched. will do that shortly.

File: app/templates/components/media/media-information.hbs
Comment (trmcnvn) [inline app/templates/components/media/media-information.hbs]: `media` context needs to be removed here.

**Review by trmcnvn — CHANGES_REQUESTED** [review]
:+1: Will merge after final changes.

File: app/templates/components/media/media-information.hbs
Comment (matthewdias) [reply app/templates/components/media/media-information.hbs]: done 😁

**Comment by matthewdias** [comment]
@vevix all ready 😄

**Review by trmcnvn — APPROVED** [review]

**Pull request merged**
